tests/testthat/test_aggregate.R

```diff
@@ -0,0 +1,101 @@
+test_that("aggregation: medianPolish", {
+    ## numerical example taken from ?stats::medpolish
+    x <- rbind(c(14,15,14),
+               c( 7, 4, 7),
+               c( 8, 2,10),
+               c(15, 9,10),
+               c( 0, 2, 0))
+    colnames(x) <- LETTERS[1:3]
+    rownames(x) <- paste0("pep", 1:5)
+    x2 <- medianPolish(x)
+    expect_is(x2, "numeric")
+    expect_identical(length(x2), ncol(x))
+    mp <- stats::medpolish(x, trace.iter = FALSE)
+    ## Check decomposition
+    expect_identical(x, mp$overall + outer(mp$row, mp$col, "+") + mp$residuals)
+    expect_identical(names(x2), colnames(x))
+})
+
+
+test_that("aggregation: robustSummary", {
+    ## numeric example taken from `MSnbase::combineFeatures` on
```

Comment:
Do you want to check for the errors here as well (`is.null(colnames(x))`)?

Reply:
Done